site/usage/embed.md

```diff
@@ -35,15 +36,45 @@ The next step is to create a DOM element that the visualization will be attached
 Then use Vega-Embed's provided function to embed your spec.
 
 ```js
-// optional argument passed to Vega-Embed to specify vega-lite spec. More info at https://github.com/vega/vega-embed
-var opt = {
-  "mode": "vega-lite"
-};
-
-vega.embed("#vis", vlSpec, opt, function(error, result) {
-  // Callback receiving the View instance and parsed Vega spec
-  // result.view is the View, which resides under the '#vis' element
-});
+// More argument info at https://github.com/vega/vega-embed
+vega.embed("#vis", yourVlSpec);
 ```
 
 Vega-Embed automatically adds links to export an image, view the source, and open the specification in the online editor. These links can be individually disabled. For more information, read the [Vega-Embed documentation](https://github.com/vega/vega-embed).
+
+Here is the fianl html file in the easiest way to embed Vega-Lite:
+```html
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Embedding Vega-Lite</title>  
```

Comment:
Above, we show how to use cdnjs, which I think should be the default for beginners.